common/src/main/resources/wynntils.mixins.json

```diff
@@ -8,11 +8,14 @@
     "BossHealthOverlayMixin",
     "ClientPacketListenerMixin",
     "ConnectionMixin",
+    "ConnectScreenMixin",
     "MinecraftMixin",
     "MultiPlayerGameModeMixin",
+    "OptionsMixin",
     "ScoreboardMixin",
     "ScreenMixin",
-    "ConnectScreenMixin"
+    "KeyMappingMixin",
+    "accessors.OptionsAccessor"
```

Comment:
@Incompleteusern To keep this list sane, I've tried to keep it sorted alphabetically. I see you fixed ConnectScreenMixin, but KeyMappingMixin ended up last...

Reply:
Okay, Ill probably fix that later

Reply:
Done